### ISSUE TITLE
When running tests locally, NVDA depends on the system language to be English

### ISSUE THREAD

**Issue by MarcoZehe** [nvaccess/nvda#13362]
### Steps to reproduce:

1. On a system that is not set to English as the language, but has a language supported by NVDA, build NVDA from source.
2. Run tests, at least the system tests.

### Actual behavior:

Almost all tests fail because the NVDA test harness expects English language output everywhere, including the Chrome UI. Local tests cannot be run and fail almost 100%.

### Expected behavior:

NVDA should force itself and the apps it uses to run in English if it needs that, even on non-English systems.

#### NVDA installed/portable/running from source:

Running from source.

#### NVDA version:

Any current master that supports running tests.

#### Windows version:

Any supported.

#### Name and version of other software in use when reproducing the issue:

Chrome browser for system robot Chrome tests.

#### Does the issue still occur after restarting your computer?

Yes.

#### Have you tried any other versions of NVDA? If so, please report their behaviors.

No.

#### If NVDA add-ons are disabled, is your problem still occurring?

Yes.

#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?

N/A.

**Comment by CyrilleB79** [comment]
> Expected behavior:
> NVDA should force itself and the apps it uses to run in English if it needs that, even on non-English systems.

While I agree that being able to run system tests on a non English environment would be appreciable, I think it is not realistic:
* forcing NVDA to run in English is probably doable with the new `--lang` command flag
* Forcing the app to run in English may add extra steps and it should then be guaranteed that the initial language of the app is restored after the test even if test fails: this begin to be very difficult.
* I think changing Windows UI in English requires to restart the computer or the session; if this is confirmed, I do not see how automated tests can do it.

Maybe an alternative Expected behavior could be to force NVDA to run in English and tag the tests that depends on an external English UI input.
Then add a command to run only the subset of tests that do not depend on English UI. This would allow to run a subset of the tests on a non-english system. Better than nothing.
